src/braket/pulse/pulse_sequence.py

```diff
@@ -331,6 +332,71 @@ def _format_parameter_ast(
             return _FreeParameterExpressionIdentifier(parameter)
         return parameter
 
+    def _parse_arg_from_calibration_schema(
+        self, argument: Dict, waveforms: Dict[Waveform], frames: Dict[Frame]
+    ) -> Any:
+        nonprimitive_arg_type = {
+            "frame": getattr(frames, "get"),
+            "waveform": getattr(waveforms, "get"),
+            "expr": FreeParameterExpression,
+        }
+        if argument["type"] in nonprimitive_arg_type.keys():
+            return nonprimitive_arg_type[argument["type"]](argument["value"])
+        else:
+            return __builtins__[argument["type"]](argument["value"])
```

Comment:
We shouldn't be using `__builtins__` since this is an internal implementation.

Reply:
Fixed